assets/dev/js/frontend/handlers/base-carousel.js

```diff
@@ -133,6 +133,9 @@ export default class CarouselHandlerBase extends SwiperHandlerBase {
 				this.a11ySetPaginationTabindex();
 				this.handleElementHandlers();
 			},
+			init: () => {
```

Comment:
Could you please check if this issue is related as well: https://elementor.atlassian.net/browse/ED-11434?